chain-signatures/contract/src/config.rs

```diff
@@ -1,23 +1,46 @@
+use std::collections::HashMap;
+
 use borsh::{self, BorshDeserialize, BorshSerialize};
-use serde::{Deserialize, Serialize};
+use near_sdk::serde::{Deserialize, Serialize};
 
-#[derive(Clone, Debug, Serialize, Deserialize, BorshSerialize, BorshDeserialize, PartialEq, Eq)]
+#[derive(
+    Clone, Default, Debug, Serialize, Deserialize, BorshSerialize, BorshDeserialize, PartialEq, Eq,
+)]
 pub struct Config {
-    /// Timeout for triple generation in milliseconds.
-    pub triple_timeout: u64,
-    /// Timeout for presignature generation in milliseconds.
-    pub presignature_timeout: u64,
-    /// Timeout for signature generation in milliseconds.
-    pub signature_timeout: u64,
-}
-
-impl Default for Config {
-    fn default() -> Self {
-        Self {
-            triple_timeout: min_to_ms(20),
-            presignature_timeout: secs_to_ms(30),
-            signature_timeout: secs_to_ms(30),
-        }
+    #[serde(flatten)]
+    pub entries: HashMap<String, DynamicValue>,
+}
+
+impl Config {
+    pub fn get(&self, key: &str) -> Option<&serde_json::Value> {
```

Comment:
No, dynamic value is purely for internal representation. This is because `serde_json::Value` does not implement borsh serialization. Dynamic value is just a wrapper to facilitate our storage of the config in contract state